.github/workflows/test.yml

```
@@ -27,6 +27,24 @@ jobs:
         name: ic-hs-test
         authToken: '${{ secrets.CACHIX_AUTH_TOKEN }}'
 
+      # Free up disk space on Ubuntu
+    - name: Free Disk Space (Ubuntu)
+      uses: jlumbroso/free-disk-space@main
+      if: matrix.os == 'ubuntu-latest'
```

```suggestion
      if: startsWith(matrix.os, 'ubuntu-')
```
just a nit, in case we want other versions in the matrix sometime...